src/utility.jl

```diff
@@ -0,0 +1,28 @@
+function unwrap!{T <: AbstractFloat}(ϕ::Array{T}; discont=π)
+  """Unwrap angles by changing deltas between values to 2π complement.
+
+  Unwrap radian phase ϕ by changing absolute jumps greater than `discont` to
+  their 2π complement.
+
+  Args:
+    ϕ: Input array.
+    discont: Maximum discontinuity between values. Optional, default is π.
+
+  Returns:
+    out: Output array of unwrapped phases.
+
+  See also:
+    numpy.unwrap()
+  """
+  Δ = diff(ϕ)
+  Δmod = mod(Δ + π, 2 * π) - π
+  Δmod[(Δmod .== -π) & (Δ .> 0)] = π
```

Comment:
It feels brittle to test for equality with an irrational number. But, let's try it and see how it goes.